tests/translator/input/error_resource_properties_not_dict.yaml

```diff
@@ -0,0 +1,4 @@
+Resources:
+  Function:
+    Type: AWS::Serverless::Function
+    Properties:
```

Comment:
What if `Properties` doesn't exist? Does this cover this case as well? Or is that already caught?

Reply:
I think that is already covered and it was just this case that wasn't.